### DISCUSSION TITLE
Loud click at start of all TTS results

### DISCUSSION THREAD

**Discussion by retry-the-user** [deepgram/community#855] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

I made a straight copy of Orion and Luna saying the same text you use on your doc pages.  I wanted to compare the natural tonality to your samples.  But every single TTS result I get has this loud, audible click at the start. I will include the samples.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

I am using the following params apart from model selection: encoding=linear16&sample_rate=16000

### If possible, please attach an example audio file to reproduce the issue.

[R.zip](https://github.com/user-attachments/files/16275261/R.zip)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@retry-the-user   If you are hearing clicking sounds we recommend adding `container=none` to your request to prevent request header information being misinterpreted as audio, which can result in static or click sounds. Give this a try and see if improves at all.


You can read more about this API Parameter here: https://developers.deepgram.com/docs/tts-encoding

**Comment by retry-the-user** [comment]
Cool. That seems to have fixed it.
